iothub/device/src/Common/Exceptions/DeviceMessageLockLostException.cs

```diff
@@ -8,10 +8,14 @@
 namespace Microsoft.Azure.Devices.Client.Exceptions
 {
     /// <summary>
-    /// The exception that is thrown when an attempt to communicate with a device fails because the lock token was lost (if the connection is lost and regained for example). This timeout has the same effect as if the message was abandonned.
+    /// The exception that is thrown when an attempt to communicate with a device fails because the lock token was lost
+    /// (if the connection is lost and regained for example). This timeout has the same effect as if the message was abandonned.
```

Comment:
"This timeout has the same effect as if the message was abandoned." - what timeout? (also, typo with "abandonned").

How about we call out - This exception is thrown when attempting to reject/abandon/complete a cloud-to-device message with a lock token that has already expired. This can happen if you reject/abandon/complete a message after 1 minute of receiving it, or your client connection was lost and regained receiving the message but before you could reject/abandon/complete it.

Reply:
"This timeout has the same effect as if the message was abandoned." - This was already present. I don't have full context so left it as is. If you have better knowledge on this, I can update.

Reply:
As of now,  I'm adding what you suggested above instead.

Reply:
FYI - It does not timeout in 1 minute so I don't think we should specify that value.

Reply:
I think the timeout statement was probably copied over from somewhere. My best is that it meant that even if you abandon a message, if the lock duration has already timed out then this exception will be thrown.